frontend/src/components/workspaces/Templates/TemplatesTable/TemplatesTable.tsx

```diff
@@ -91,34 +91,29 @@ const TemplatesTable: FC<ITemplatesTableProps> = ({ ...props }) => {
           });
     };
    */
-  const [expandedId, setExpandedId] = useState([
-    window.sessionStorage.getItem('prevExpandedIdDashboard') ?? '',
-  ]);
+  const [expandedId, setExpandedId] = useState(
+    window.sessionStorage.getItem('prevExpandedIdDashboard')?.split(',') ?? []
+  );
 
   const expandRow = (
     activeInstances: number,
     rowId: string,
     create: boolean
   ) => {
     if (activeInstances) {
-      let expandedIdTmp: string;
-      if (!create)
-        expandedId[0] === rowId
-          ? (expandedIdTmp = '')
-          : (expandedIdTmp = `${rowId}`);
-      else expandedIdTmp = `${rowId}`;
-      setExpandedId([expandedIdTmp]);
+      expandedId.includes(rowId)
+        ? setExpandedId(old => (create ? old : old.filter(id => id !== rowId)))
+        : setExpandedId(old => [...old, `${rowId}`]);
     }
   };
 
   useEffect(() => {
-    window.sessionStorage.setItem('prevExpandedIdDashboard', `${expandedId}`);
+    window.sessionStorage.setItem(
+      'prevExpandedIdDashboard',
+      `${expandedId.toString()}`
```

Comment:
```suggestion
      expandedId.join(',')
```
Always better to keep parsing and marshalling symmetrical.